R/apply_fmt_fn.R

```diff
@@ -89,7 +89,11 @@ alias_as_fmt_fn <- function(x, call = parent.frame()) {
     return(label_cards(digits = decimal_n, scale = scale, width = width))
   }
 
-  cli::cli_abort("Formatting functions/aliases must be a function, a non-negative integer, or a formatting string, e.g. {.val xx.x}.", call = call)
+  cli::cli_abort(
```

Comment:
@ddsjoberg is the intention here to actually error out or to pass it into the error argument as well when running `apply_fmt_fn()`